client/components/button/style.scss

```diff
@@ -115,7 +115,8 @@ button {
 // Primary buttons
 .button.is-primary {
 	background: var( --color-accent );
-	border-color: darken( $blue-medium, 8% );
+	// border-color: darken( $blue-medium, 8% );
```

Comment:
Any specific reason why we'd want to leave this in as a comment?

Reply:
Only so that I can reference it in https://github.com/Automattic/wp-calypso/pull/29399#discussion_r241349367